attribution/api_attribution.py

```diff
@@ -111,17 +112,17 @@ async def compute_attributions(
     async def hierarchical_perturbation(
         self,
         original_input: str,
-        init_chunk_size: int,
+        init_chunk_size: Optional[int] = None,
         stride: Optional[int] = None,
         perturbation_strategy: PerturbationStrategy = FixedPerturbationStrategy(),
-        attribution_strategies: list[str] = ["cosine", "prob_diff"],
+        attribution_strategies: list[str] = ["prob_diff"],
         threshold_attribution_strategy: Optional[str] = None,
         static_threshold: Optional[float] = None,
         use_absolute_attribution: bool = False,
         unit_definition: Literal["token", "word"] = "token",
         ignore_output_token_location: bool = True,
         logger: Optional[ExperimentLogger] = None,
-        verbose: bool = False,
+        verbose: int = 0,
```

Comment:
Why was this changed from `bool` to `int`?

If the goal is to be able to set whether the output should be verbose or not, I think that a `bool` would be better.

If we want to be able to set multiple levels of verbosity (0 being none, 1 more verbose, and potentially 2 and above being even more verbose), then I think an `int` would be fine, but maybe the argument name should be `verbosity`, and we should fix the docstring of this function and its usage in `hierarchical_perturbation.ipynb`